### PR TITLE
labs nav treatment - READY FOR REVIEW

### PR DESCRIPTION
**Pull request by owocki**
makes gitcoin labs/research would look like as a first class product offering

slack convo https://gitcoincore.slack.com/archives/CCZFZDT9V/p1542127912012200

screen shot 2018-11-13 at 12 09 09 pm
screen shot 2018-11-13 at 12 08 55 pm

why we should make this change:

> i do think that research/labs is a more abstract product offering than our others… but it nicely encapsulates our “ship fast” /  ‘provide value quickly’ ethos, and it also provides a brand halo effect around us in that we’re doing bleeding edge stuff.  so as a web3 person, theres stuff to LEARN about advanced things at gitcoin, not just work to work on or people to give work to.

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Yes I like it \M/

**Comment by mbeacom** [comment]
Looks fine to me! Should we cap the topnav following the full implementation of this change, though? I'm all for this, but my main concern is the topnav mushrooming and ultimately losing value if we add too much. 🤔

**Comment by owocki** [comment]
@octavioamu would it be possible to whip up a quick on-hover animation for the gitcoin labs logo, similar to what we have for gitcoin/bounties/kudos ?

**Comment by octavioamu** [comment]
Sure, I will take the svg and make some
Image of Yaktocat

**Comment by owocki** [comment]
thanks @octavioamu  -- i want to get @PixelantDesign 's feedback after she's had the option to sleep on it

**Comment by owocki** [comment]
> Looks fine to me! Should we cap the topnav following the full implementation of this change, though? I'm all for this, but my main concern is the topnav mushrooming and ultimately losing value if we add too much. 🤔

What do you mean 'cap' the top nav?  Sure, lets call the topnav capped at 5-6 sections;  Should be easy to adhere to for the next several months as our roadmap involves iterating on kudos/grants/bounties/labs for the next 6 months and not launching major new products

and beyond that, i know that @PixelantDesign has a nice accordian dropdown nav mocked up, which IIRC breaks down the products into a 'develop', 'contribute', and 'maintain' step, and can scale to more products if needed.

**Comment by owocki** [comment]
@octavioamu @PixelantDesign feedback on https://github.com/gitcoinco/web/issues/2821 ?

**Comment by thelostone-mc** [comment]
should we build this out after we get #2821 out?
@mbeacom / @owocki

**Comment by owocki** [comment]
@thelostone-mc yes i dont think it makes sense to merge this before the design is done

File: app/assets/v2/images/top-bar/gitcoin_labs_white.svg [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mbeacom) [inline app/assets/v2/images/top-bar/gitcoin_labs_white.svg]: All of these svgs need optimized/squashed.

**Pull request closed without merging**